src/Components/RosTable/RenderColumn.js

```diff
@@ -49,9 +51,22 @@ export const displayOS = (data) => {
 };
 
 export const displayLastReported = (data) => {
+    const daysAgoSeven = moment().subtract(7, 'days');
+    const isStale = moment(data).isBefore(daysAgoSeven);
+    const exactDate = dateStringByType('exact')(new Date(data));
+    const relativeDate = dateStringByType('relative')(new Date(data));
+    // eslint-disable-next-line max-len
+    const staleTooltipText = `System was not refreshed in the last 7 days.\nSuggestions for the system might be outdated due to reporting issues.\nLast reported: ${exactDate}`;
+
     return (
         data === null ?
             <span>{ NO_DATA_VALUE }</span> :
-            <DateFormat date={ data } />
+            isStale ?
+                <Tooltip content={<div>{ staleTooltipText }</div>}>
+                    <span className='staleText'>
+                        <ExclamationTriangleIcon color='#f09800' size='sm'/> {relativeDate}
```

Comment:
ditto

```suggestion
                        <ExclamationTriangleIcon color="var(--pf-global--warning-color--100)" size='sm'/> {relativeDate}
```

Reply:
@kgaikwad updated and pushed the changes 🚀